### PR TITLE
Use bplog to send to master operation to delete from sc_history

### PR DESCRIPTION
**Pull request by adizaimi**
- Use bplog commands to delete from sc_history
   - Remove remnants of stale osql_check_replicant_numops tunable
   - sc_rebuilds.test can send send trim command to any node, fix tail -n +3

### CODE REVIEW

File: sqlite/src/comdb2build.c [hunk 1]
```diff
@@ -6602,3 +6632,43 @@ void comdb2_drop_view(Parse *pParse, SrcList *pName)
 out:
     free_schema_change_type(sc);
 }
+
+/* delete entry by seed from comdb2_sc_history 
+ * called from stored procedure function db_comdb_delete_sc_history()
+ */
+int comdb2DeleteFromScHistory(char *tablename, uint64_t seed)
+{
+    BpfuncArg *arg = (BpfuncArg*) malloc(sizeof(BpfuncArg));
```
Comment (riverszhang89) [inline sqlite/src/comdb2build.c:6641]: A structure initialized on stack would suffice.

File: db/bpfunc.c [hunk 2]
```diff
@@ -520,3 +526,12 @@ static int exec_rowlocks_enable(void *tran, bpfunc_t *func, struct errstat *err)
     return rc;
 }
 
+static int exec_delete_from_sc_history(void *tran, bpfunc_t *func,
+                                       struct errstat *err)
+{
+    BpfuncDeleteFromScHistory *tblseed = func->arg->tblseed;
+    int rc = bdb_del_schema_change_history(NULL, tblseed->tablename, tblseed->seed);
```
Comment (markhannum) [inline db/bpfunc.c:533]: Why aren't you using the transaction for bdb_del_schema_change_history here?

Comment (adizaimi) [reply db/bpfunc.c]: Good catch. Fixing.

File: sqlite/src/comdb2build.c (hunk 1 above)
Comment (adizaimi) [reply sqlite/src/comdb2build.c:6641]: yes, i think on the stack works. thanks.

**Comment by markhannum** [comment]
Looks good- okay to merge!

**Pull request merged**